ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityTextQuestionBlock.ts

```diff
@@ -11,12 +11,27 @@ export class ActivityTextQuestionBlock extends ActivityQuestionBlock<string> {
     public confirmEntry: boolean;
     public confirmPrompt: string | null = null;
     public mismatchMessage: string | null = null;
+    public confirmationValue: string | null;
 
     constructor() {
-        super();
+      super();
     }
 
     public get questionType(): QuestionType {
-        return QuestionType.Text;
+      return QuestionType.Text;
+    }
+
+    public get answer(): string | null {
+        return this._answer;
+    }
+
+    public set answer(value: string | null) {
+        this.setAnswer(value);
+    }
+
+    setAnswer(value: string | null, doValidation: boolean = true): void {
+      this._answer = value;
+      this.confirmEntry && (this.confirmationValue = value);
+      doValidation && this.validate();
     }
```

Comment:
our "blocks" only take a single answer value, but we need to somehow initialize the confirmationValue .